### PR TITLE
Exclude energy producers from EB methods for EU datasets

### PR DESCRIPTION
**Pull request by kaskranenburgQ**
This PR updates the data input method for PP/CHP/HP from the entso: EB() route to the standard data input route. 
In addition, it performs a data migration on all relevant datasets for these keys.
For now, the commit message for the migration is : "Updating Data extraction method" --> Feel free to change this message if you feel that this is not descriptive enough

### CODE REVIEW

**Comment by mabijkerk** [comment]
@kaskranenburgQ / @kndehaan did any of you check this migration by exporting the EU datasets to ETSource from this branch and verify that nothing changes?

**Comment by kaskranenburgQ** [comment]
@mabijkerk I did, except from some rounding differences in each dataset, there are no changes.

File: db/migrate/20241008145926_e_bkiller/commits.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+
+
+# If you are creating only a single commit, you may instead use a single :all
+# field to have all of the CSV columns be included. This cannot be used when
+# creating multiple commits.
+- fields:
+  - :all
+  message:
+    Updating Data extraction method
```
Comment (mabijkerk) [inline db/migrate/20241008145926_e_bkiller/commits.yml:10]: Can we make this more informative? Something to indicate that we exclude energy production keys from the EB method for EU datasets and set them through migration instead - while keeping the data values itself the same.

**Review by mabijkerk — CHANGES_REQUESTED** [review]

**Comment by mabijkerk** [comment]
Note: the full load hours were already excluded from the EB method. They therefore do not need to be migrated.

**Comment by mabijkerk** [comment]
Note: heat import keys were already excluded from the EB method. They therefore do not need to be migrated.

**Pull request merged**